api/app/utils/fba_calculator.py

```diff
@@ -457,7 +457,7 @@ def get_critical_hours(  # pylint: disable=too-many-arguments
     if critical_ffmc == 0.0 and resulting_hfi >= target_hfi:
         logger.info('All hours critical for HFI %s. FFMC %s has HFI %s',
                     target_hfi, critical_ffmc, resulting_hfi)
-        return '13:00 - 7:00'
+        return (13.0, 7.00)
```

Comment:
Can we just create the `CriticalHoursHFI` object here to remove the extra tuple step later on?